migration.go

```diff
@@ -157,6 +202,39 @@ func (m Migrations) Current() *Migration {
 	return m[len(m)-1]
 }
 
+// AppliedAfter selects the applied migrations from a Store after a given version.
+func AppliedAfter(store Source, source Source, version int64) (Migrations, error) {
+	var appliedAfter Migrations
+	appliedMigrations, err := store.Collect()
+	if err != nil {
+		return nil, err
+	}
+
+	found := false
+	for _, migration := range appliedMigrations {
+		if migration.Version == version {
+			found = true
+			break
+		}
+		appliedAfter = append(appliedAfter, migration)
+	}
```

Comment:
Do we sort here? If not, we implicitly rely on the db's internals and that's not guaranteed to be stable - you might append migrations that were applied before the one you're looking for